client/js/actions/banks.js

```diff
@@ -11,9 +11,9 @@ const types = [
 export const Constants = wrapper(types, []);
 
 
-export function getBankHierarchy(){
+export function getBankHierarchy(payload) {
```

Comment:
This update reflects the actual structure of this action.  Other code was dispatching the action directly, without calling this function, and that code has been changed to use the function now.